pingdom-checks/developer-portal.ping

```diff
@@ -0,0 +1,15 @@
+# -*- sh -*-
+
+VA_DEVELOPER_PORTAL_BACKEND_SLACK_ID="$SLACK_LIGHTHOUSE_ALERTS"
+
+pingdom save-check \
+  --template https-public-200 \
+  -a name=production-developer-portal-backend-health \
+  -a host=api.va.gov \
+  -a url="/internal/developer-portal/public/ping" \
```

Comment:
This path makes me feel sad... "internal" and "public"? Which is it, internal or public?